src/common/io.ts

```diff
@@ -578,7 +579,7 @@ export class IOReadable implements AoIORead {
 
 export class IOWriteable extends IOReadable implements AoIOWrite {
   protected declare process: AOProcess;
-  private signer: ContractSigner;
+  private aoSigner: AoSigner;
```

Comment:
going to ask to revert just to `signer` - should it ever change we don't have to modify the attribute name

Reply:
[3b23f80](https://github.com/ar-io/ar-io-sdk/pull/175/commits/3b23f80581ea331b89ac31756421f74799315e2c)